src/pages/index.tsx

```diff
@@ -147,6 +149,9 @@ const Home: NextPage<HomePageProps> = ({ storefront }) => {
   })
 
   const { control, watch } = useForm<NFTFilterForm>({})
+  const [showProfilePopover, setShowProfilePopover] = useState(false)
```

Comment:
https://www.radix-ui.com/docs/primitives/components/popover

Radix has a popover can you look to use it?